### PR TITLE
TH-1028 | Use new ids params to get list of events

### PR DESCRIPTION
**Pull request by Sartonon**
## Description :sparkles:

- Use new ids param to get list of events

## Issues :bug:

**[TH-1028](https://helsinkisolutionoffice.atlassian.net/browse/TH-1028):** 

## Additional notes :spiral_notepad:

### CODE REVIEW

**Review by nikomakela — DISMISSED** [review]
Looks good to me.

**Comment by Sartonon** [comment]
Deployment is not working. I think build passes?

File: src/__tests__/eventIntegration.test.ts [hunk 1]
```diff
@@ -104,9 +104,17 @@ it('resolves eventsByIds correctly', async () => {
   const publisherId = 'publisherId';
 
   const getMock = jest.fn().mockResolvedValue({
-    id: eventId,
-    publisher: publisherId,
-  } as EventDetails);
+    data: [
```
Comment (nenonja) [inline src/__tests__/eventIntegration.test.ts:107]: no new eventList test? or does this change cover it?

Comment (nenonja) [reply src/__tests__/eventIntegration.test.ts:107]: Ok, we dont do any change to events-helsinki-ui, we just change this old one to use new api-endpoint.

**Review by nenonja — APPROVED** [review]
Looks good. Build was red but I re-run the job and now all  ✅

File: src/__tests__/eventIntegration.test.ts (hunk 1 above)
Comment (Sartonon) [reply src/__tests__/eventIntegration.test.ts:107]: yeah

**Pull request merged**